src/tools/rust-analyzer/crates/ide-assists/src/handlers/convert_named_struct_to_tuple_struct.rs

```diff
@@ -150,6 +150,7 @@ fn edit_struct_references(
     }
 }
 
+#[allow(unnecessary_block)]
```

Comment:
where exactly is the unnecessary block in this function?

Reply:
`match_ast!{}` macro, somehow
in `_ => {}`

Reply:
line number 205